release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -419,6 +419,24 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[id="vSphere-deprecated-configuration-parameters"]
+==== Deprecated VMware vSphere configuration parameters
+
+{product-title} {product-version} deprecates the following vSphere configuration parameters. You can continue to use these parameters, but the installation program does not automatically specify these parameters in the `install-config.yaml` file.
+
+* `platform.vsphere.vCenter`
+* `platform.vsphere.username`
+* `platform.vsphere.password`
+* `platform.vsphere.datacenter`
+* `platform.vsphere.defaultDatastore`
+* `platform.vsphere.cluster`
+* `platform.vsphere.folder`
+* `platform.vsphere.resourcePool`
+* `platform.vsphere.apiVIP`
+* `platform.vsphere.ingressVIP`
+* `platform.vsphere.MachinePlatform`
```

Comment:
remove `MachinePlatform` that isn't deprecated

Reply:
Good catch. I removed this from the list.

Reply:
Hi @jcpowermac . I moved the note to a more suitable section. Please let me know what you think.